### ISSUE TITLE
Support for Forms 3.0?

### ISSUE THREAD

**Issue by czuck** [jsmarcus/Xamarin.Plugins#116]

**Comment by Ali-YousefiTelori** [comment]
after several days there is not update,i have to fork and change it manually 👎

**Comment by czuck** [comment]
I just found this .. https://github.com/jsmarcus/Iconize

**Comment by xufeipyxis** [comment]
Yes, Xam.FormsPlugin.Iconize is not needed to use this plugin in a Xamarin.Forms project any more.
